packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -160,18 +160,38 @@ export class Listbox extends FASTElement {
     protected focusAndScrollOptionIntoView(): void {
         if (this.contains(document.activeElement) && this.firstSelectedOption) {
             this.firstSelectedOption.focus();
-            this.firstSelectedOption.scrollIntoView({ block: "nearest" });
+            requestAnimationFrame(() => {
+                this.firstSelectedOption.scrollIntoView({ block: "nearest" });
```

Comment:
I've occasionally seen some unexpected scrolls of parent scrolling containers with scrollIntoView and switched to a more targeted `  this.menuElement.scrollTo(0, focusedOption.offsetTop);` in my picker prototype.  Not sure if that's an issue here.